index.html

```diff
@@ -147,18 +180,33 @@ <h2>
         <h3>
           Get current position
         </h3>
-        <p>
-          Request the user's current location. If the user allows it, you will
-          get back a position object.
-        </p>
+        <aside class="addition" id="a5">
+          <span class="marker">Candidate Addition:</span> Add the descripton
+          and update examples for AccuracyMode.
+        </aside>Request the user's current location. If the user allows it, you
+        will get back a position object. <ins cite="#a5">The website can
+        influence the position's accuracy by using the `accuracyMode` option.
```

Comment:
Make this more specific on how to set accuracyMode (and link to the accuracyMode attribute definition):

> The website can request an [=approximate position=] by passing a {{PositionOptions}} with its {{PositionOptions/accuracyMode}} attribute set to {{AccuracyMode/"precise"}}.